src/js/jsx/search/SearchBar.jsx

```diff
@@ -416,10 +424,17 @@ define(function (require, exports, module) {
 
             var filters = this.state.filter,
                 filtersString,
-                idSplit = id.split("-"),
-                // Possible current types are Menu_Command, Current_Doc, Recent_Doc, All_Layers
-                type = idSplit[0],
-                category = "category-" + type;
+                idSplit,
+                type;
+
+            if (id) {
```

Comment:
This is weird. Above, we check for `id === null`. I'm not even sure what type `id` is supposed to have here because it's undocumented but I guess it can be some other falsey value like `undefined`? In any case, this smells funny.